script.js

```diff
@@ -0,0 +1,22 @@
+// select the form/input/result
+
+//Ideas for how to create list?
+// tasks could be added to an array
+//(array.push to add and array.splice to remove?)
+// (this could be a table? Each time we add something we add 
+// a <tr>? Or could declare variable n and every time
+// submit is pressed, add to <tr>[n] then n+1)
+
+//Function for adding tasks
+// when submit is pressed add it to list 
+
+//Function to remove tasks
+
+//Function for complete tasks
+// When complete, get added to different list/ gets a .hidden 
```

Comment:
We could make each task an object, with one of its properties being "status"

That would allow us to:

- run `filter()` to show custom views and hide completed task
- easily expand metadata to include things like "importance"

Reply:
Ooooh I LOVE the object idea!